overlord/snapstate/check_snap.go

```diff
@@ -542,7 +542,7 @@ func checkAndCreateSystemUsernames(si *snap.Info) error {
 
 	// then create
 	// TODO: move user creation to a more appropriate place like "link-snap"
-	extrausers := !release.OnClassic
+	extrausers := !release.OnClassic || release.OnTouch
```

Comment:
Could we look for the extrausers database instead?

Reply:
Is the reason to avoid having `OnTouch` altogether? Because further changes implemented in the Touch fork of snapd check for `OnTouch` and decide whether to fill in the new AARE templates or not. Our diverse range of maintainer ports don't support these well enough in the case of Docker, which makes sense to have on a tablet designated for development purposes.